packages/api-gateway/__tests__/places/getPlaces.test.js

```diff
@@ -0,0 +1,16 @@
+const { fastify } = require('../../src/drivers/http/server');
+
+// At least in my pc it lasts almost 15s
+jest.setTimeout(120000);
```

Comment:
@Alejandrocuartas why tests are taking all this time? it doesn't make sense because we have few tests to execute, so I would say that maybe there is something that is not working properly and I would suggest to take a look there

Reply:
Santiago, actually, it takes about 40s in my pc, but i put it in 2m for avoiding the timeout. I also use a local real database for the tests.

Reply:
This is the behavior on my pc.
![Screen Shot 2022-04-12 at 21 21 38](https://user-images.githubusercontent.com/28733681/163099864-d74f10ad-2ffe-4e29-821e-ffc1df0500c9.png)

Reply:
I think that logs and swagger are reducing the performance a little bit, this is what I did to solve that issue:
- https://github.com/Platzi-Master-C9/booking-services/pull/123/files#diff-b00e6b54dba26cd350d6b2c3ab7e8eec355e340ae8fb14d12953694b1639a040R15
- https://github.com/Platzi-Master-C9/booking-services/pull/123/files#diff-b00e6b54dba26cd350d6b2c3ab7e8eec355e340ae8fb14d12953694b1639a040R56
![Screen Shot 2022-04-12 at 21 28 09](https://user-images.githubusercontent.com/28733681/163100244-13341ee2-0891-4332-a9da-b06d4a8e5e2a.png)

Reply:
@Alejandrocuartas I noticed that other tests are failing because of timeout, so I added a ridiculous timeout to solve this issue for now until we found what is causing the poor performance.... so you can rebase your PR and get rid of this line.

PR: https://github.com/Platzi-Master-C9/booking-services/pull/173

Reply:
Thanks @JoseNoriegaa, i did it.